examples/chat/module/rust/src/lib.rs

```diff
@@ -34,7 +32,12 @@ type AdminId = Vec<u8>;
 struct Node {
     room_to_admin: HashMap<RoomId, AdminId>,
     admin_to_room: HashMap<AdminId, RoomId>,
```

Comment:
Could potentially use a `bimap`?

Reply:
Ooh, looks neat, I didn't know about that.  Left as a TODO for now though.